cypress/fixtures/designSystem/flatTable.json

```diff
@@ -1,8 +1,8 @@
-{ 
-    "captionOtherLanguage": {
-        "caption": "mp150ú¿¡üßä"
-      },
-      "captionSpecialCharacter": {
-        "caption": "!@#$%^*()_+-=~[];:.,?{}&\"'<>"
-  } 
+{
+  "captionOtherLanguage": {
+    "captionSpecialCharacters": "otherLanguage"
```

Comment:
Should this be `"captionOtherLanguage"` rather than `"captionSpecialCharacters"`?